at-hoangtran/Week3/Day3/Exercise6-Day2.rb

```diff
@@ -0,0 +1,16 @@
+arr = [1, 2, 3, 1]
+# Give an array and print sum of array
+puts arr.sum
+
+# Create an array with elements from 0 to 1000 that can be divided by 3 and 5
+a = (1..100).select { |i| (i % 3).zero? && (i % 5).zero? }
+puts a
+
+# Give an array and remove odd numbers
+arr.select! { |i| i.even? }
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/a5ed3ba5b6a204f5dbf8b2b54972d06d02f2dcdb/at-hoangtran/Week3/Day3/Exercise6-Day2.rb#L10"></span>Style/SymbolProc: Pass <code>&amp;:even?</code> as an argument to <code>select!</code> instead of a block.</td>
    </tr>
  </tbody>
</table>